### PR TITLE
Feature/m0078 ddg 429 UI pack 13.1.0

### PR DESCRIPTION
**Pull request by gbalan-microej**

### CODE REVIEW

File: PlatformDeveloperGuide/uiImageGenerator.rst [hunk 1]
```diff
@@ -100,7 +102,7 @@ To do that the Image Generator provides some services to implement. This chapter
          <publications/>
          
          <dependencies>
-            <dependency org="com.microej.pack.ui" name="ui-pack" rev="13.0.0">
+            <dependency org="com.microej.pack.ui" name="ui-pack" rev="[UI Pack version]|">
```
Comment (apoudret-microej) [inline PlatformDeveloperGuide/uiImageGenerator.rst:105]: remove `|` ?

File: PlatformDeveloperGuide/uiSimulation.rst [hunk 2]
```diff
@@ -68,18 +70,172 @@ The Front Panel extension does not provide any widgets. Some compatible widgets
...
+
+* The simple or double buffering (default value): ``doubleBufferFeature=true|false``.
+* The backlight (enabled by default): ``backlightFeature=true|false``.
+* The non-rectangular displays: ``filter="xxx.png"``. Some displays can have another appearance (for instance: circular); the Front Panel can simulate using a filter (see :ref:`section_frontpanel_widget`). This filter defines the pixels inside and outside the whole display area. The filter image must have the same size as the rectangular display area. A display pixel at a given position will not be rendered if the pixel at the same position in the mask is fully transparent.
+* The :ref:`standard<display_pixel_structure_standard>` pixel formats.
+* The :ref:`driver-specific<display_pixel_structure_driver>` pixel formats: ``extensionClass="xxx"``. This class must be added in the Front Panel project and implement the interface ``ej.fp.widget.Display.DisplayExtension``.
+
+.. _section_ui_simulation_customdrawing:
+
+Custom Drawings
```
Comment (apoudret-microej) [inline PlatformDeveloperGuide/uiSimulation.rst:111]: Extract `Custom Drawings` and `New Drawings` sections to separate tutorials?

File: ApplicationDeveloperGuide/UI/MicroUI/fontDesigner.rst [hunk 3]
```diff
@@ -128,13 +128,26 @@ character size in pixels.
...
+The styles options ``Built-in`` tag the font as bold, italic, or bold and italic. This style can be retrieved by the MicroEJ Application thanks the methods ``Font.isBold()`` and ``Font.isItalic()``. Adjust the styles options according to the font:
+
+* The font is a `plain` font: select ``None`` option for each style.
+* The font is a `bold` font: select ``Built-in`` for the style `bold` and ``None`` for the other styles.
+* The font is an `italic` font: select ``Built-in`` for the style `italic` and ``None`` for the other styles.
+* The font is a `bold` and `italic` font: select ``Built-in`` for the styles `bold` and `italic` and ``None`` for ``underline``.
 
-Font Designer features three drop-downs, one for each of ``BOLD``, ``ITALIC``
-and ``UNDERLINED``. Each drop-down has three options: ``None``, ``Built-in`` and ``Dynamic``. Use only ``None`` option. Otherwise an error at MicroEJ application compiletime will occur (incompatible font file).
+.. warning:: When a font holds a ``Built-in`` style or when the style `underline`, an error at MicroEJ application compile-time is thrown (incompatible font file).
```
[the remark this answers is not in the document]
Comment (gbalan-microej) [reply ApplicationDeveloperGuide/UI/MicroUI/fontDesigner.rst:150]: you're right: the right sentence is: 

.. warning:: When a font holds a ``Dynamic`` style or when the style `underline` is not ``None``, an error at MicroEJ application compile-time is thrown (incompatible font file).

File: PlatformDeveloperGuide/uiSimulation.rst [hunk 4]
```diff
@@ -68,18 +70,172 @@ The Front Panel extension does not provide any widgets. Some compatible widgets
...
-than display rectangular area. A display pixel at a given position will
-be not rendered if the pixel at the same position in mask is fully
-transparent.
+The widget Display implements the interface ``ej.microui.display.LLUIDisplayImpl`` to be compatible with the implementation of the MicroUI class `Display <https://repository.microej.com/javadoc/microej_5.x/apis/ej/microui/display/Display.html>`_.
+
+This widget manages:
+
+* The simple or double buffering (default value): ``doubleBufferFeature=true|false``.
+* The backlight (enabled by default): ``backlightFeature=true|false``.
+* The non-rectangular displays: ``filter="xxx.png"``. Some displays can have another appearance (for instance: circular); the Front Panel can simulate using a filter (see :ref:`section_frontpanel_widget`). This filter defines the pixels inside and outside the whole display area. The filter image must have the same size as the rectangular display area. A display pixel at a given position will not be rendered if the pixel at the same position in the mask is fully transparent.
```
[the remark this answers is not in the document]
Comment (gbalan-microej) [reply PlatformDeveloperGuide/uiSimulation.rst:105]: A pixel in the mask may be partially transparent; the pixel in the skin will be rendered

**Pull request merged**